io.github.aginies.watermark.yml

```diff
@@ -0,0 +1,38 @@
+app-id: io.github.aginies.watermark
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+command: watermark_app
+tags:
+  - utility
+  - image-processing
+desktop-file-name-suffix: (Flatpak)
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --filesystem=host
+  - --talk-name=org.freedesktop.Notifications
+  - --share=network
+modules:
+  - name: watermark_app
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 watermark_app.py /app/bin/watermark_app
+      - install -Dm644 io.github.aginies.watermark.png /app/share/icons/hicolor/256x256/apps/io.github.aginies.watermark.png
+      - install -Dm644 io.github.aginies.watermark.desktop /app/share/applications/io.github.aginies.watermark.desktop
+    sources:
+      - type: file
+        path: watermark_app.py
+      - type: file
+        path: io.github.aginies.watermark.png
+      - type: file
+        path: io.github.aginies.watermark.desktop
+  - name: python-dependencies
```

Comment:
1. dependencies are always installed first.  It's logical.
2. build must be offline. This isn't.

Reply:
this should be addressed now.